src/common/tests/btree.c

```diff
@@ -704,8 +704,10 @@ ik_btr_batch_oper(void **state)
 	}
 
 	D_ALLOC_ARRAY(arr, key_nr);
-	if (arr == NULL)
+	if (arr == NULL) {
 		fail_msg("Array allocation failed");
```

Comment:
I did not find definition of fail_msg() will this function quit program as well? or it just output message.

Reply:
According to cmocka manual, fail_msg() forces the test to fail immediately and quit, printing the reason, so I think it's not necessary to call 'return' herel.

Reply:
So coverity is not smart enough, we could mark coverity as negative or return here to make it happy.